.eslintignore

```diff
@@ -1,3 +1,4 @@
 /coverage
 /third_party
 /public/js/gulliver.js
+/public/js/ligthouse-chart.js
```

Comment:
Oh, I see. I think the right thing to do is tell eslint `google` is a global via a comment. Actually, also needs the `browser` environment to get away with `XMLHttpRequest`:

```js
/* global google */
/* eslint-env browser */
```